addons/binding/org.openhab.binding.vallox/src/main/java/org/openhab/binding/vallox/handler/ValloxSerialHandler.java

```diff
@@ -0,0 +1,404 @@
+                break;
+            case IncommingCurrent:
+                state = new DecimalType(vs.incommingCurrent);
+                break;
+            case InEfficiency:
+                state = new DecimalType(vs.inEfficiency);
+                break;
+            case InputFanStopThreshold:
+                state = new DecimalType(vs.inputFanStopThreshold);
+                break;
+            case IoPortMultiPurpose1:
+                state = new DecimalType(vs.ioPortMultiPurpose1);
+                break;
+            case IoPortMultiPurpose2:
+                state = new DecimalType(vs.ioPortMultiPurpose2);
+                break;
+            case LastErrorNumber:
+                state = new DecimalType(vs.lastErrorNumber);
+                break;
+            case MaxSpeedLimitMode:
+                state = getOnOff(vs.maxSpeedLimitMode);
+                break;
+            case OutEfficiency:
+                state = new DecimalType(vs.outEfficiency);
+                break;
+            case PostHeatingOn:
+                state = getOnOff(vs.postHeatingOn);
+                break;
+            case PowerState:
+                state = getOnOff(vs.powerState);
+                break;
+            case PreHeatingOn:
+                state = getOnOff(vs.preHeatingOn);
+                break;
+            case PreHeatingSetPoint:
+                state = new DecimalType(vs.preHeatingSetPoint);
+                break;
+            case Program:
+                state = new DecimalType(vs.program);
+                break;
+            case Program2:
+                state = new DecimalType(vs.program2);
+                break;
+            case RadiatorType:
+                state = getOnOff(vs.radiatorType);
+                break;
+            case SelectStatus:
+                state = new DecimalType(vs.selectStatus);
+                break;
+            case ServiceReminder:
+                state = new DecimalType(vs.serviceReminder);
+                break;
+            case ServiceReminderIndicator:
+                state = getOnOff(vs.serviceReminderIndicator);
+                break;
+            case SupplyFanOff:
+                state = getOnOff(vs.supplyFanOff);
+                break;
+            case TempExhaust:
+                state = new DecimalType(vs.tempExhaust);
+                break;
+            case TempIncomming:
+                state = new DecimalType(vs.tempIncomming);
+                break;
+            case TempInside:
+                state = new DecimalType(vs.tempInside);
+                break;
+            case TempOutside:
+                state = new DecimalType(vs.tempOutside);
+                break;
+            default:
+                logger.warn("Got update notification for unknown vallox property: {}", channel);
+                break;
+        }
+        // XXX: future improvement: check whether value really has changed to reduce amount of updates
```

Comment:
I removed the XXX due to an forums discussion from which I learn that the optimzation is not necessary because it's already done in the internal channel handling